content/trial_getting_started/examples/openstack_hello_world_example.md

```diff
@@ -7,6 +7,172 @@ alwaysopen = false
 
 {{%children style="h2" description="true"%}}
 
-## Step 1: Install the Cloudify Manager as a Docker container
 
-The easiest way to deploy Cloudify trial manager is as a Docker container. This tutorial assumes that you have [Docker](https://docs.docker.com/install) installed on your local machine.
+This Example demonstrates a simple topology setup in **Openstack**, the deployment consists of a VM, a simple web service + app, and all of the essential peripherals in Openstack (security group, network, etc.)
+
+Cloudify allows for multiple user interfaces. In this tutorial we will demonstrate the usage of the Cloudify management console (web UI) and the Cloudify command line (cfy). The following steps demonstrate both approaches.
+
+## Step 1: Install the Cloudify Manager inside Docker container
+
+In order to deploy the Cloudify manager inside Docker container follow the instructions in [this page]({{< relref "trial_getting_started/trial_install.md" >}}).
+
+## Step 2: Create the secrets containing Openstack credentials
+
+To connect to Openstack a set of credentials are required. Cloudify recommends storing such sensitive information in a Cloudify secret. Secrets are kept encrypted in a secure way and used in run-time by the system. learn more about Cloudify secrets [here]({{< relref "/cli/orch_cli/secrets.md" >}}).
+
+Store the secrets in the manager:
+
+This can be done through command line or directly via Cloudify management console.
+
+From the hosting shell run:
+```bash   
+docker exec -it cfy_manager_local sh -c "cfy secrets create openstack_username --secret-string <user_name>"
+docker exec -it cfy_manager_local sh -c "cfy secrets create openstack_password --secret-string <password>"
+docker exec -it cfy_manager_local sh -c "cfy secrets create openstack_tenant_name --secret-string <tenant_name>"
+docker exec -it cfy_manager_local sh -c "cfy secrets create openstack_auth_url --secret-string <url>"
+
+```                                             
+openstack_username - OS_USERNAME as specified in Openstack RC file.
+
+openstack_password - Openstack user password.
+
+openstack_tenant_name - OS_TENANT_NAME as specified in Openstack RC file.
+
+openstack_auth_url - OS_AUTH_URL as specified in Openstack RC file. For this example use v2.0 authentication url. 
+
+_**Tip**: Running commands on Docker containers can be applied directly from the hosting shell by encapsulating the command in quotes and using the docker exec command. for example: `docker exec -it <container name> sh -c "<the command>"`.  Alternatively, you can open a shell directly in the container by executing: `docker exec -it <container image name> /bin/bash`_
```

Comment:
Since we already display these commands with docker exec do we need to tell them this?

Reply:
I think it is worth keeping it. Since "Docker" is not native to all users, and since we pretty much force them to run Cloudify as Docker container, I think we should give them some basic tips..